source/tablist.rst

```diff
@@ -0,0 +1,33 @@
+==================
+PlayerList/Tablist
+==================
+
+Adventure only supports changing the header(encased in red) and footer(encased in yellow) of the tablist.
+
+.. image:: /images/tablist.png
```

Comment:
images should be in the same directory, or in a `tablist/` directory

sphinx will separate things at build-time